src/test/java/teammates/test/cases/browsertests/AdminEmailPageUiTest.java

```diff
@@ -59,6 +59,7 @@ private void testCompose() throws Exception {
 
         ______TS("send email - no subject");
```

Comment:
@xpdavid have already started working on it, just waiting for the issue to be created.
There is a small typo in AdminEmailComposeSendAction.java which needs to be corrected i.e reciver->receiver, should  I correct that in this pr only or create a new issue for that?